### PR TITLE
Add autoHidePlayer prop

### PR DESCRIPTION
**Pull request by hungvu193**
Sometimes, When I drag the slider on IOS, it will be disappeared due to controlsTimer hide animation (default 200ms).
This prop will prevent it.

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -89,8 +89,10 @@ const VideoPlayer = (tempProps) => {
                 }
                 controlsTimer = null;
             }, 2000);
+     
         }
     };
+    };
```
Comment (ihmpavel) [inline dist/index.js:95]: This looks like a duplicated line by mistake

File: dist/index.js [hunk 2]
```diff
@@ -78,7 +78,7 @@ const VideoPlayer = (tempProps) => {
         else if (controlsState === ControlStates.Visible) {
             hideAnimation();
         }
-        if (controlsTimer === null) {
+        if (controlsTimer === null && props.autoHidePlayer) {
```
Comment (ihmpavel) [inline dist/index.js:81]: If you want to change/add something, update please Typescript files located under `lib` folder.

In your case `props.ts` and `index.ts`. After that run `yarn build`. That will generate JS files in `/dist` folder and TS definitions. Update also props documentation in `README.md`

I'll be happy to accept this MR

**Review by ihmpavel — CHANGES_REQUESTED** [review]
Looks good, but you should edit files in `/lib` folder

File: dist/index.js (hunk 2 above)
Comment (hungvu193) [reply dist/index.js:81]: got it, let me update my PR

**Comment by hungvu193** [comment]
@ihmpavel let have a look, I have just updated the PR 😄

**Pull request merged**